services-js/commissions-search/src/FacetList.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import "./FacetList.css";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class FacetList extends Component {
+  render() {
+    const GET_AREAS = gql`
+      {
+        policyTypes {
+          id
+          name
+        }
+      }
+    `;
+
+    function Area(props) {
+      return (
+        <label className="cb">
+          <input
+            id="checkbox-area"
```

Comment:
`id` needs to be unique on the page if it’s provided, but you might not need it if you’re wrapping in a `label`.

Reply:
done